filter.go

```diff
@@ -1,20 +1,16 @@
 package lnroll
 
 import (
-	"errors"
 	"fmt"
 	"time"
 
 	"github.com/apg/ln"
+	"github.com/pkg/errors"
+	"github.com/stvp/roll"
 )
 
-type Client interface {
-	Critical(err error, extras map[string]string) (uuid string, e error)
-	Error(err error, extras map[string]string) (uuid string, e error)
-}
-
 // New returns a new FilterFunc which reports errors to Rollbar.
-func New(client Client) ln.FilterFunc {
+func New() ln.FilterFunc {
```

Comment:
We shouldn't change the interface here. It may seem weird to take an interface when we can simply call roll directly, but having to setup module level `roll.Token` is not so great.